### ISSUE TITLE
Does not work with PHP 8.0

### ISSUE THREAD

**Issue by yasuhirokimura** [RainLoop/rainloop-webmail#2040]
OS: FreeBSD 12.1-RELEASE amd64
PHP: PHP 8.0.0 Release Candidate 2 (installed by using FreeBSD ports collection)
Rainloop: v1.14.0 community edition (installed by using FreeBSD ports collection)

I tried Rainloop with PHP 8.0 but following notice and error messages are displayed.

> Notice: spl_autoload_register(): Argument #2 ($do_throw) has been ignored, spl_autoload_register() will always throw in /usr/local/www/rainloop/rainloop/v/1.14.0/app/handle.php on line 63
>
> Notice: spl_autoload_register(): Argument #2 ($do_throw) has been ignored, spl_autoload_register() will always throw in /usr/local/www/rainloop/rainloop/v/1.14.0/app/libraries/MailSo/MailSo.php on line 30
>
> Fatal error: Array and string offset access syntax with curly braces is no longer supported in /usr/local/www/rainloop/rainloop/v/1.14.0/app/libraries/RainLoop/Actions.php on line 9263

**Comment by the-djmaze** [comment]
Correct, already solved.

TL;DR:
https://github.com/RainLoop/rainloop-webmail/pull/1982

**Comment by yasuhirokimura** [comment]
@the-djmaze Is there plan of new version release?

**Comment by the-djmaze** [comment]
@yasuhirokimura i have no idea, i'm not the owner of this project.
I only have my own fork which is heavily modified.

**Comment by deg-pl** [comment]
@RainLoop when the code will support PHP 8.0? I think it is important not only for me...

**Comment by c-burrows** [comment]
All my other web apps work well with PHP 8, so, having to keep PHP 7.4 installed on my server because of Rainloop is niggling to say the least.

**Comment by yasuhirokimura** [comment]
@deg-pl @c-burrows v1.15.0 is released on January 17th and it supports PHP 8.

**Comment by c-burrows** [comment]
Great work, however, sieve doesn't work with PHP 8.

Is this a new issue or is it related to this one?
